### ISSUE TITLE
UI : code given in the Argilla starting page for logging data has "y" which causing invalid syntax

### ISSUE THREAD

**Issue by Jitendra-Dash** [recognai/rubrix#1811]
**Describe the bug**
Before creating a dataset, a code is provided to log data into argilla using python in the argilla UI. In that code, there is a "y" letter which is of no use and throws error.

**To Reproduce**
1. Go to the 'Argilla Start page without any dataset'
2. you will see a code for logging data using python
3. use the same code 

**Expected behavior**
Invalid syntax

**Screenshots**

**Environment (please complete the following information):**
 - OS [e.g. iOS]:
 - Browser [e.g. chrome, safari]:
 - Argilla Version [e.g. 1.0.0]:
 - ElasticSearch Version [e.g. 7.10.2]:
 - Docker Image (optional) [e.g. argilla:v1.0.0]:

**Additional context**

**Comment by dvsrepo** [comment]
Thanks so much for reporting, this will be fixed ASAP

**Comment by dvsrepo** [comment]
@leiyre this is the same as #1784 (closed in favour of this one)

I will take this opportunity to add a more meaningful example (#1801)

Replace the one we have by:

```python
# install datasets library with pip install datasets
import argilla as rg
from datasets import load_dataset

# load dataset from the hub
dataset = load_dataset("argilla/gutenberg_spacy-ner", split="train")

# read in dataset, assuming its a dataset for token classification
dataset_rg = rg.read_datasets(dataset, task="TokenClassification")

# log the dataset
rg.log(dataset_rg, "gutenberg_spacy-ner")
```

**Comment by leiyre** [comment]
Hello @Jitendra-Dash, the issue has already been fixed, you can update to version `1.0.1`. 
https://github.com/argilla-io/argilla/releases/tag/v1.0.1, 
Thank you
